capa/features/common.py

```diff
@@ -92,23 +97,32 @@ def __nonzero__(self):
 
 
 class Feature(abc.ABC):
-    def __init__(self, value: Union[str, int, float, bytes], description=None):
+    def __init__(
+        self, value: Union[str, int, float, bytes], access: Optional[str] = None, description: Optional[str] = None
+    ):
         """
         Args:
           value (any): the value of the feature, such as the number or string.
           description (str): a human-readable description that explains the feature value.
         """
         super(Feature, self).__init__()
-        self.name = self.__class__.__name__.lower()
+
+        if access is not None:
+            if access not in VALID_ACCESS:
+                raise ValueError("access '%s' must be one of %s" % (access, VALID_ACCESS))
+            self.name = self.__class__.__name__.lower() + "/" + access
```

Comment:
not sure if `.name` should include the trailing access or if it should just be considered during hash & str.

it probably doesn't matter and this might be nit picking

Reply:
good point - I've moved this to `__str__`